README.md

```diff
@@ -45,6 +45,8 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [DuckDuckGo](https://duckduckgo.com/)
 - [Swisscows](https://swisscows.com/en)
 - [Naver](https://www.naver.com/)
+- [Ecosia](https://www.ecosia.org)
```

Comment:
ecosia is just a web portal using bing, we can remove it

Reply:
Done

Reply:
it's not

Reply:
sorry my bad, I think everything is done now.